src/platform/OpenThread/OpenThreadUtils.cpp

```diff
@@ -126,7 +126,8 @@ void LogOpenThreadStateChange(otInstance * otInst, uint32_t flags)
         }
 #if CHIP_CONFIG_SECURITY_TEST_MODE
         {
-            const otNetworkKey * otKey = otThreadGetNetworkKey(otInst);
+            otNetworkKey * otKey;
+            otThreadGetNetworkKey(otInst,otKey);
```

Comment:
Was this tested?  Because I would expect this to be an exploitable security bug right here...

Reply:
Is the security bug regarding printing the key ?The two statements after the changes?

Reply:
No, it's about the misuse of the `otThreadGetNetworkKey` API.

Again: was this tested?

Reply:
Updated to use the local variables instead of the pointers. Locally tested and shows the network key in the DUT logs.